recipes/df2gspread/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "df2gspread" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "98501399986871495d115b2331110fef576658b81002c95393c7ea478cb87c84" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - argparse_dep.patch
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
It appears that only `python` and `setuptools` are required in the build section since you're using `--single-version-externally-managed`. Thus I don't think you need to patch the `setup.py` for `argparse`.